web/impact/impact/settings.py

```diff
@@ -131,6 +122,20 @@ class Base(Configuration):
         'impact.middleware.MethodOverrideMiddleware',
     ]
 
+    if os.environ.get('READ_REPLICA_DATABASE_URL'):
+        DATABASES.update(values.DatabaseURLValue(  # pragma: no cover
+            alias='read-replica',
+            environ_name='READ_REPLICA_DATABASE_URL').value)
+
+        SLAVE_DATABASES = ['read-replica']  # pragma: no cover
```

Comment:
FWIW I'd prefer we used "primary/replica" terminology in our codebase instead of "master/slave".